onnx/defs/rnn/defs.cc

```diff
@@ -19,22 +19,30 @@ void RNNShapeInference(InferenceContext& ctx) {
   if (hidden_size_value > 0)
     hidden_size.set_dim_value(hidden_size_value);
 
+  auto batch_major_value = getAttribute(ctx, "batch_major", 0);
```

Comment:
nit: If the final name of the `batch_major` or `batch_axis` flag is changed, it may be helpful to update the name of this variable.